pkg/operator/operator.go

```diff
@@ -157,6 +158,7 @@ func NewOperator(ctx context.Context, operator *operator.Operator) (context.Cont
 		operator.Elected(),
 		kubeDNSIP,
 		clusterEndpoint,
+		nil,
```

Comment:
If we're passing through `nil` all of the time on startup, why do we even need this as part of our constructor?